changelog_unreleased/typescript/pr-9162.md

```diff
@@ -0,0 +1,16 @@
+#### Variable declarations with union types can line break after the equals sign (#9162 by @ehrencrona)
+
+<!-- prettier-ignore -->
+```typescript
+// Input
+let a: LoooooooooooooooongTypeNames | ThatMakeTheLineGrowOverEightyChars = a.shortFn();
+
+// Prettier stable
+let a:
+  | LoooooooooooooooongTypeNames
+  | ThatMakeTheLineGrowOverEightyChars = a.shortFn();
+
+// Prettier master
+let a: LoooooooooooooooongTypeNames | ThatMakeTheLineGrowOverEightyChars =
+  a.shortFn();
+```
```

Comment:
Why we need break them if they fit in one line?

Reply:
I'm not sure I understand. The line as a whole (`let ... shortFn();`) is 88 characters and therefore needs a line break.

Reply:
Yes, we usually breaks here, but in some cases we keep line more than 80 characters for readable, in `stable` branch we have mixed content for our eyes:
 
> | LoooooooooooooooongTypeNames

only type

> | ThatMakeTheLineGrowOverEightyChars = a.shortFn();

type + call expression